transcom/mymove
# INT B-18739 and B-18740 ppm aoa and packet download from multi move dashboard

## [Agility ticket B-18740](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18740)
## [Agility ticket B-18739](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18739)

## Summary

Adding the PPM Packet & AOA Packet download functionality to the multi move dashboard. Since these packets are available at the shipment level, the customer will need to expand the move in order to see the download button, but ain't no thing.

I wasn't super hot on how the whole dropdown button looked inside the shipment container, so I updated the dropdown button to include a `minimal` parameter that removes the outline & box shadow of the button, which really cleaned it up nicely.

This PR does the following:
- updates `ButtonDropdown` component to take in `minimal` parameter
- added flash messaging capabilities to the Multi-Move dashboard
- added conditional logic that renders the PPM packet download button
- added logic that determines which buttons the dropdown shows based on `ppmShipment` values
- updated css to use `@media` that changes flex direction on mobile
- added to multi move test data to handle PPM download button use cases
- updated all relevant tests based on updated code

**NOTE**
I am also throwing in some swagger code that was removed (IT WAS ME OKAY) on accident [in this commit](https://github.com/transcom/mymove/commit/8e20a63d3df787213319af4f2a705d9364e3de1f). This commit is already in `main` so just ensuring it is back in `integrationTesting`.


### How to test

1. Access MM as a customer
2. Go through the process of creating a move with a couple of PPM shipments, sign it
3. Go through the SC process and approve
4. Go through the TOO process and make approvals
5. Sign back in as the customer and you should now see the download button visible on the shipment


## Screenshots
![Screenshot 2024-04-05 at 1 36 21 PM](https://github.com/transcom/mymove/assets/136510600/46241395-a75a-4edb-a094-8b50fa82d848)

![Screenshot 2024-04-05 at 1 36 44 PM](https://github.com/transcom/mymove/assets/136510600/3396b953-e98d-478d-bb6d-f2117e090c77)

![Screenshot 2024-04-05 at 1 37 04 PM](https://github.com/transcom/mymove/assets/136510600/1a478626-3738-4289-bc63-07fba1387779)

![Screenshot 2024-04-05 at 1 36 54 PM](https://github.com/transcom/mymove/assets/136510600/ac2ed9a1-e5af-4301-b913-f0253c2ee3b5)

![Screenshot 2024-04-05 at 2 20 25 PM](https://github.com/transcom/mymove/assets/136510600/923200af-981d-4e63-afb0-c8108ceb4777)
